service/gradle/dependencies.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
   implementation group: "org.apache.commons", name: "commons-dbcp2"
   implementation group: "org.apache.commons", name: "commons-lang3"
   implementation group: "org.apache.commons", name: "commons-pool2"
+  implementation group: "org.apache.logging.log4j", name: "log4j-api"
+  implementation group: "org.apache.logging.log4j", name: "log4j-to-slf4j"
```

Comment:
interesting - I don't see any changes to the actual log4j dependencies we're pulling in in the lockfile so I don't know what this is changing. where was the error you saw showing up?